package.json

```diff
@@ -20,6 +20,8 @@
   },
   "dependencies": {
     "@openzeppelin/contracts": "^2.5.0",
+    "@openzeppelin/contracts-ethereum-package": "^2.5.0",
+    "@openzeppelin/upgrades": "^2.8.0",
```

Comment:
These two packages I had to install as described here: https://www.notion.so/How-To-test-2215ca8fb6354f8183d1ea70a65f9f2d. Not sure if it is necessary to push this change in package.json. I can remove it.